code/game/objects/effects/anomalies/anomalies_bluespace.dm

```diff
@@ -2,9 +2,9 @@
 /obj/effect/anomaly/bluespace
 	name = "jumper"
 	icon_state = "bluespace"
-	desc = "A mysterious anomaly that causes teleportation around it."
+	desc = "A hole in the fabric of bluespace, perforating reality around it."
```

Comment:
Kinda meh on this change

Reply:
I mean. What else teleports things lol. I think its fine for it to be blue-space.